website/docs/r/monitor_activity_log_alert.html.markdown

```diff
@@ -64,9 +64,9 @@ resource "azurerm_monitor_activity_log_alert" "main" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the activity log alert. Changing this forces a new resource to be created.
+* `name` - (Required) The name of the activity log alert. The `Name` field can only contain `alphanumeric characters`, `hyphens`, `underscores`, `periods`, and `parentheses`. Changing this forces a new resource to be created.
```

Comment:
since we don't normally give detailed validation information in the docs description unless it's a list of possible values, should we remove this?

```suggestion
* `name` - (Required) The name of the activity log alert. Changing this forces a new resource to be created.
```

Reply:
Fixed.